### PR TITLE
[Method References] LPS-162265 Convert method references to field references in class extends BaseJSPProductNavigationControlMenuEntry

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I replaced method getServletContext with method getRequestDispatcher in class BaseJSPProductNavigationControlMenuEntry.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1565

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701  , Could you please help me check the test results when they finish ? Thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

**Review by dantewang — APPROVED** [review]

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/product/navigation/control/menu/FragmentEntriesGeneratorProductNavigationControlMenuEntry.java [hunk 1]
```diff
@@ -86,12 +87,11 @@ public boolean isShow(HttpServletRequest httpServletRequest)
 	}
 
 	@Override
-	@Reference(
-		target = "(osgi.web.symbolicname=com.liferay.fragment.web)",
-		unbind = "-"
-	)
-	public void setServletContext(ServletContext servletContext) {
-		super.setServletContext(servletContext);
+	protected RequestDispatcher getRequestDispatcher(String jspPath) {
```
Comment (tinatian) [inline modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/product/navigation/control/menu/FragmentEntriesGeneratorProductNavigationControlMenuEntry.java:90]: @W2jx Please make the method to be getServletContext, then you can move the duplicate logic _servletContext.getRequestDispatcher(jspPath); into its base class

Comment (W2jx) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/product/navigation/control/menu/FragmentEntriesGeneratorProductNavigationControlMenuEntry.java:90]: I will fix it.

**Pull request closed without merging**